.github/workflows/release_workflow.yml

```
@@ -0,0 +1,59 @@
+# Give your test a name!
+name: ReleaseWorkflow
+
+ runs every two months on the 25th around 1300
+on:
+  schedule:
+#    - cron: '00 13 25 */2 *'# Do we want it to run at the start or end of every month?
+    - cron: '*/3 * * * *'
+    # minute (0 - 59), hour (0 - 23), day of month (1 - 31), month(1 - 12 or JAN-DEC), day of week (0 - 6 or SUN-SAT)
+#on:
+#  push:
+#    branches:
+#      - main
+# Set up the Continuous Integration job
+jobs:
+  release-workflow:
+    runs-on: ubuntu-latest
+    # This section collects together the steps involved in running the test
+    steps:
+      - name: Checkout branch
+        uses: actions/checkout@v2
+
+      - name: Update CITATION.cff file
+        working-directory: ./tests
+        run: |
+          python release_workflow.py
+
+      - name: Validate a CITATION.cff from a subdirectory
+        uses: citation-file-format/cffconvert-github-action@2.0.0
+        with:
+          args: "--infile ./CITATION.cff --validate"
+
+      - name: Convert CITATION.cff to Zenodo metadata format
+        uses: citation-file-format/cffconvert-github-action@2.0.0
+        with:
+          args: "--infile ./CITATION.cff --format zenodo --outfile ./.zenodo.json"
+
+      - name: Commit + push citation.cff and Zenodo metadata
+        run: |
+          git config --global user.name 'timothy22000'
+          git config --global user.email 'timothy22000@gmail.com'
+          git add .zenodo.json CITATION.cff
+          git commit -m "Automated update of citation.cff and Zenodo metadata"
+          git push
```

```suggestion
      # https://github.com/marketplace/actions/add-commit
      - name: Commit + push citation.cff and Zenodo metadata
        uses: EndBug/add-and-commit@v9
        with:
          add: '[".zenodo.json", "CITATION.cff"]'
          message: "Automated update of citation.cff and Zenodo metadata"
          new_branch: release-workflow-bot
          author_name: "CI User"
          author_email: "ci-user@github.local"
          committer_name: "CI User"
          committer_email: "ci-user@github.local"
```

We can use an action to do the add/commit/push for us and then don't have to worry about providing the correct token.